DevIncubator-Autopark/MyQueue.cs

```diff
@@ -0,0 +1,105 @@
+﻿namespace DevIncubatorAutopark
+{
+    internal class MyQueue<T>
+    {
+        private const int DefaultSize = 10;
+        private T[] queueArr;
+        private int length;
+        private int startIndex;
+        private int endIndex;
+
+        public MyQueue()
+        {
+            queueArr = new T[DefaultSize];
+        }
+
+        public MyQueue(int length)
+        {
+            if (length < 0)
+            {
+                throw new ArgumentException("Размер очереди не может быть меньше 0", nameof(length));
+            }
+            queueArr = new T[length];
+            this.length = length;
```

Comment:
Rename private fields starting with `_` a and then you don't need to use a keyword `this`

Reply:
fixed